packages/client/src/views/SysAdmin/Config/Forms/Wizard/FormConfigWizard.tsx

```diff
@@ -215,22 +216,28 @@ export function FormConfigWizard() {
           version: version + 1
         })}
         pageIcon={<></>}
-        topBarActions={[
-          <TertiaryButton
-            id="settings"
-            key="settings"
-            icon={() => <SettingsBlue />}
-            onClick={() => dispatch(goToFormConfigWizard(event, 'settings'))}
-          ></TertiaryButton>,
-          <PrimaryButton
-            key="save"
-            size="small"
-            disabled={status === ActionStatus.PROCESSING}
-            onClick={() => setStatus(ActionStatus.MODAL)}
-          >
-            {intl.formatMessage(buttonMessages.save)}
-          </PrimaryButton>
-        ]}
+        topBarActions={
+          !location.pathname.includes('settings')
+            ? [
+                <TertiaryButton
+                  id="settings"
+                  key="settings"
+                  icon={() => <SettingsBlue />}
+                  onClick={() =>
+                    dispatch(goToFormConfigWizard(event, 'settings'))
+                  }
+                ></TertiaryButton>,
+                <PrimaryButton
+                  key="save"
+                  size="small"
+                  disabled={status === ActionStatus.PROCESSING}
+                  onClick={() => setStatus(ActionStatus.MODAL)}
+                >
+                  {intl.formatMessage(buttonMessages.save)}
+                </PrimaryButton>
+              ]
+            : []
+        }
```

Comment:
Hiding the save and settings button on settings page.  Changes take effect immediately